corehq/apps/locations/models.py

```diff
@@ -17,12 +17,19 @@
 from corehq.apps.domain.models import Domain
 from corehq.apps.products.models import SQLProduct
 from corehq.toggles import LOCATION_TYPE_STOCK_RATES
+from corehq.util.soft_assert import soft_assert
 from mptt.models import MPTTModel, TreeForeignKey, TreeManager
 
 
 LOCATION_REPORTING_PREFIX = 'locationreportinggroup-'
 
 
+def notify_of_deprecation(msg):
+    _assert = soft_assert(notify_admins=True, fail_if_debug=True)
+    message = "Deprecated Locations feature used: {}".format(msg)
+    _assert(False, message)
```

Comment:
I've got a few things on the chopping block, so I figured I'd make this generic enough to use also in those other places when their time comes